### PR TITLE
post message for balanced open requests

### PR DESCRIPTION
**Pull request by niteshbalusu11**
Posts a message when a balanced channel open request is received.

Files changed:
interface/icons.json
post/get_received_message.js
post/post_settled_invoice.js

### CODE REVIEW

**Comment by niteshbalusu11** [comment]
https://github.com/alexbosworth/balanceofsatoshis/issues/233

File: post/get_received_message.js [hunk 1]
```diff
@@ -197,9 +232,18 @@ module.exports = ({description, lnd, payments, received}, cbk) => {
       // Final received message
       message: [
         'messageDetails',
+        'parseBalancedOpen',
         'receivedMessage',
-        ({messageDetails, receivedMessage}, cbk) =>
+        ({messageDetails, parseBalancedOpen, receivedMessage}, cbk) =>
       {
+        if (!!parseBalancedOpen) {
+          const message = `Received a ${escape(parseBalancedOpen.capacity)} Balanced Channel Open request from ${escape(parseBalancedOpen.alias)} \`${parseBalancedOpen.destination}\``;
```
Comment (alexbosworth) [inline post/get_received_message.js]: Potentially the alias is empty, could be `alias || id.substring(0, 8)` or something like that?

Comment (niteshbalusu11) [reply post/get_received_message.js]: fixed it.

**Comment by alexbosworth** [comment]
merged it

**Pull request closed without merging**